iwyu_globals.cc

```diff
@@ -94,6 +94,8 @@ static void PrintHelp(const char* extra_msg) {
          "   --no_comments: do not add 'why' comments.\n"
          "   --no_fwd_decls: do not use forward declarations.\n"
          "   --verbose=<level>: the higher the level, the more output.\n"
+         "   --quoted_headers_first: when sorting includes, place quoted\n"
```

Comment:
"Quoted" is a quality of the `#include` line, not of the header.

All the other `LineSortOrdinal`s talk about "headers", but I think it makes sense to name this separately. The others use all sorts of heuristics to identify the kind of header, but this is really about the include line.

So I'd suggest `--quoted_includes_first`, and a thorough rename :-/

Reply:
Done.